bin/i18n/sync-codeorg-in

```diff
@@ -7,57 +7,106 @@
 
 require File.expand_path('../../../pegasus/src/env', __FILE__)
 require 'fileutils'
+require 'json'
 require 'yaml'
 require 'tempfile'
 
+require_relative 'utils'
+
 def pull_in_files_for_translation
   localize_instructions
-  exec "bin/i18n-codeorg/in.sh"
+  run_bash_script "bin/i18n-codeorg/in.sh"
 end
 
-def find_instruction(f)
-  pattern = /^\s*"instructions": (".*?"),?\n$/
-  f.each_line do |line|
-    matches = line.match pattern
-    if matches
-      return matches.captures.first
-    end
-  end
-  nil
-end
-
-def copy_instructions_to_yml(instructions)
-  File.open("dashboard/config/locales/instructions.en.yml", "w+") do |f|
-    f.write(({ "en" => { "data" => { "instructions" => instructions }}}).to_yaml)
+def copy_to_yml(label, data)
+  File.open("dashboard/config/locales/#{label}.en.yml", "w+") do |f|
+    f.write(({ "en" => { "data" => { label => data }}}).to_yaml)
     File.write(f, File.read(f))
   end
 end
 
 def reformat_quotes
-  temp_file = Tempfile.new("tempinstructions.yml")
-  File.open("dashboard/config/locales/instructions.en.yml", "r") do |f|
-    f.each_line { |line| temp_file.puts line.gsub("'\"", '"').gsub("\"'", '"').gsub("''", "'")}
+  filenames = ["instructions", "markdown_instructions"]
```

Comment:
Interesting. That could be good or bad, as there are still a few markdown instructions that have raw HTML in them and which we would probably not want to be translated. I figure that once we get it all in crowdin and see what it looks like, we can have the sanitization conversation.